src/applications/disability-benefits/all-claims/components/EducationTrainingField.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { formatReviewDate } from 'us-forms-system/lib/js/helpers';
+
+export default function EducationTrainingField({ formData }) {
+  const { name, dates } = formData;
+  let from = '';
+  let to = '';
+  if (dates) {
+    from = formatReviewDate(dates.from);
+    to = formatReviewDate(dates.to);
+  }
+
+  return (
+    <div>
+      <strong>{name}</strong>
```

Comment:
Do we need to conditional check this if it's empty so it doesn't show `undefined` or will it show blank?

Reply:
Shows blank